app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.utills.JwtUtill;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("사용자 권한 서비스 클래스")
```

Comment:
클래스를 설명할 때 "클래스"라는 단어를 쓰지 않는 것이 좋습니다. 메소드도 마찬가지에요.
```suggestion
@DisplayName("사용자 권한 서비스")
```